tests/fiaas_deploy_daemon/deployer/kubernetes/deployment/test_datadog.py

```diff
@@ -107,3 +108,17 @@ def test_adds_container_when_enabled(self, datadog, app_spec, deployment, best_e
                 'requests': {'cpu': '200m', 'memory': '2Gi'}
             }
         assert expected == deployment.as_dict()["spec"]["template"]["spec"]["containers"][-1]
+
+    def test_adds_correct_image_pull_policy_for_latest(self, config, app_spec, deployment):
+        config.datadog_container_image = CONTAINER_IMAGE_LATEST
+        datadog = DataDog(config)
+
+        datadog_spec = app_spec.datadog._replace(
+            enabled=True
+        )
+        app_spec = app_spec._replace(datadog=datadog_spec)
+
+        datadog.apply(deployment, app_spec, False)
+
+        actual = deployment.as_dict()["spec"]["template"]["spec"]["containers"][-1]
+        assert actual['image'] == CONTAINER_IMAGE_LATEST
```

Comment:
This doesn't look like it is actually testing that the image pull policy is set correctly?

Reply:
That's very true. It helps if I save the file before committing! 🙈